src/test/java/org/opensrp/web/config/SwaggerConfigTest.java

```diff
@@ -52,27 +68,40 @@ public void testApi() {
 
     @Test
     public void testGetApiInfo() {
-        assertNull(swaggerConfig.getApiInfo());
-        when(swaggerConfig.getApiInfo()).thenReturn(createTestApiInfo());
-        ApiInfo apiInfo = swaggerConfig.getApiInfo();
-        assertNotNull(apiInfo);
-        assertEquals(apiInfo.getTitle(), "Test title");
-        assertEquals(apiInfo.getDescription(), "Test description");
-        assertEquals(apiInfo.getVersion(), "VERSION 1.0");
-        assertEquals(apiInfo.getLicense(), "Test LICENSE");
-        assertEquals(apiInfo.getLicenseUrl(), "Test license url");
+        assertNotNull(swaggerConfig.getApiInfo());
```

Comment:
Why were the specific assertions for various fields of the `apiInfo` removed?

Reply:
Initially the mocked version of the class was under test as seen [here](https://github.com/opensrp/opensrp-server-web/blob/master/src/test/java/org/opensrp/web/config/SwaggerConfigTest.java#L38). So essentially the actual aclass was not being tested.

I updated the method `testGetApiInfo()` to check for null since apiInfo() is a just getter method.